src/displayapp/widgets/PopupMessage.cpp

```diff
@@ -0,0 +1,39 @@
+#include "displayapp/widgets/PopupMessage.h"
+#include <libraries/log/nrf_log.h>
+
+using namespace Pinetime::Applications::Widgets;
+
+PopupMessage::PopupMessage(const char* msg)
+  : message {msg},
+    isHidden {true},
+    btnPopup {nullptr} {
```

Comment:
please move the default values to the definition in the header, just my personal preference to keep the constructor for non-default values

Reply:
done